charts/flyte-core/templates/console/deployment.yaml

```diff
@@ -18,6 +18,10 @@ spec:
         {{- end }}
       labels: {{ include "flyteconsole.podLabels" . | nindent 8 }}
     spec:
+    {{- with .Values.flyteconsole.imagePullSecrets }}
+      imagePullSecrets:
```

Comment:
While I don't see this to be the most common scenario (hosting the flyteconsole image on a secured registry) I see it could be the case for more security-aware environments.